ledger-core/scripts/insolard/pulsar_template.yaml

```diff
@@ -31,5 +23,12 @@ metrics:
   zpagesenabled: false
   reportingperiod: 0s
 log:
-  level: debug
+  level: Debug
+  adapter: zerolog
```

Comment:
why not binlog?

Reply:
for now all components are using zerolog by default